server/room.php

```diff
@@ -383,8 +397,10 @@
     $node->{'Time'} = preg_replace('/<[^>]*>/', '', $node->{'Time'});
     $node->{'Day'} = preg_replace('/<[^>]*>/', '', $node->{'Day'});
 
-    // Remove Spaces so we can use it for a CSS class
+    // Remove special chars in Topic from XML request so we can use it for a CSS class
```

Comment:
figured this was best to include here, although I it wasnt clear if we are actually utilizing this like we are in scroll.php